test/tests/functional/pbs_server_periodic_hook.py

```diff
@@ -391,8 +391,8 @@ def test_set_as_non_admin(self):
         host_name = str(self.server.hostname)
         self.server.create_hook(hook_name, attrs={'enabled': "False"})
         attrs = {'event': "periodic", 'freq': '120'}
-        match_str1 = str(TEST_USER1) + "@" + host_name + \
-            " is unauthorized to access hooks data from server " + host_name
+        match_str1 = "unauthorized to access hooks data from server " + \
+            host_name
```

Comment:
host_name is used here

Reply:
this hostname is server  hostname. We are not using client hostname.

Reply:
This is the wrong hostname